resworb/exporter.py

```diff
@@ -18,10 +18,13 @@ def export(self, kinds: Union[str, Iterable[str]] = "all") -> Dict[str, List]:
             "bookmarks": self.get_bookmarks,
             "histories": self.get_histories,
         }
-        if kinds != "all":
-            factory = {kind: factory[kind] for kind in kinds}
+        if kinds == "all":
+            return {k: list(v()) for k, v in factory.items()}
 
-        return {k: list(v()) for k, v in factory.items()}
+        if isinstance(kinds, str):
+            kinds = [kinds]
+
+        return {kind: factory[kind] for kind in kinds}
```

Comment:
I guess you're missing something here...

Reply:
Do you mean exception handling?